### ISSUE TITLE
Provide wheel package on PyPI

### ISSUE THREAD

**Issue by edmorley** [erikrose/blessings#115]
Hi!

Please could this package also by made available as a wheel on PyPI?
https://pypi.python.org/pypi/blessings

Wheels have the following advantages:
* Faster installation time
* Allows the wheel cache to work even when using the new pip hash-checking mode
* Allows tools to statically inspect the package metadata without having to execute untrusted code.
* ...and more - see: http://pythonwheels.com

Since this package uses 2to3 on Python 3, it will need to have separate wheels for each, rather than a universal wheel:
https://packaging.python.org/tutorials/distributing-packages/#wheels

The Python packaging guide recommends using [Twine](https://github.com/pypa/twine) to upload packages:
https://packaging.python.org/tutorials/distributing-packages/#uploading-your-project-to-pypi

As such, the steps for building/uploading a new release are then:

```bash
$ pip2 install -U twine setuptools wheel
$ pip3 install -U setuptools wheel
$ rm -rf dist/ build/
$ python2 setup.py sdist bdist_wheel
# Work around https://bitbucket.org/pypa/wheel/issues/147/bdist_wheel-should-start-by-cleaning-up
$ rm -rf build/
$ python3 setup.py bdist_wheel
$ twine upload dist/*
```

The PyPI package listing will then display three entries for the new release, one with type "Source" and two of type "Python Wheel" (one for each of Python 2 and 3).

Many thanks :-)

**Comment by edmorley** [comment]
Ah I just spotted the use of 2to3, so this will need separate wheels for each of Python 2 and Python 3. I've updated the steps in the OP accordingly :-)

**Comment by jquast** [comment]
FYI the https://github.com/jquast/blessed fork uses wheel distribution, this project is not getting much attention, partly the reason for forking, sorry for any inconvenience :(

**Comment by jquast** [comment]
Related to #121

**Comment by erikrose** [comment]
Fixed in 1.6.1. Thanks, Ed!

**Comment by edmorley** [comment]
Many thanks :-)
